operator/v1/types_ingress.go

```diff
@@ -665,6 +674,22 @@ type IBMLoadBalancerParameters struct {
 	Protocol IngressControllerProtocol `json:"protocol,omitempty"`
 }
 
+// OpenStackLoadBalancerParameters provides configuration settings that are
+// specific to OpenStack load balancers.
+type OpenStackLoadBalancerParameters struct {
+	// loadBalancerIP specifies the IP address that the load balancer will use.
+	// ---
+	// Note: this field is meant to be set by the ingress controller to populate the
+	// `Service.Spec.LoadBalancerIP` field which has been deprecated in Kubernetes:
+	// https://github.com/kubernetes/kubernetes/pull/107235
+	// However, the field is still used by cloud-provider-openstack to reconcile
+	// the floating IP that we attach to the load balancer.
```

Comment:
You need to prefix these with a `+` else they'll be included in the `oc explain` output.